respec/Begrippenkader.md

```diff
@@ -1,14 +1,14 @@
 ### Opbouw beschrijvingen
-De basis voor de beschrijving van een begrippenkader is SKOS. We geven enkele aanvullingen op standaard SKOS. Zo vinden we het belangrijk dat ieder begrip een uitleg in klare taal (B2 niveau) heeft en dat ieder begrip kan worden gerelateerd aan een definitie in een formele (juridische) bron.
+Het begrippenkader is het geheel van de begripsbeschrijvingen die als één geheel worden beheerd.
 
-Voor het beschrijven van metadata hebben we gebruik gemaakt van metadatastandaarden zoals Dublin Core (dctemrs), de Provenance Ontologie (Prov-O) en ADMS/DCAT. We zien een begrippenkader niet als een DCAT dataset. DCAT is primair bedoeld om de dataset zelf te beschrijven. We zien een begrippenkader wel als een 'semantic asset' die beschreven kan worden als een ADMS dataset. Dit maakt de herkenbaarheid van de betreffende metadata beter vindbaar via het web. Ook biedt dit handvatten om deze metadatabeschrijving op te nemen in data.overheid.nl omdat ADMS een profiel op DCAT is.
+Voor het beschrijven van de metadata van een begrippenkader sluiten we aan op metadatastandaarden zoals Dublin Core (dcterms), de Provenance Ontologie (Prov-O) en ADMS/DCAT. We zien een begrippenkader als een "semantic asset": een waardevolle eenheid van geëxpliciteerde kennis. Dit komt overeen met een adms:Asset. Aangezien adms:Asset een specialisatie is van een dcat:Dataset kunnen we op deze wijze goed aansluiten op catalogi die DCAT als uitgangspunt nemen, zoals data.overheid.nl
 
-Iedere relevante klasse voor het voorliggende profiel (begrippenkader, begrip, bronverwijzing, collectie), wordt beschreven in een aparte paragraaf, inclusief de eigenschappen van die klasse. Iedere beschrijving bestaat uit 2 delen:
+Iedere relevant onderdeel voor het voorliggende profiel (begrippenkader, begrip, bronverwijzing, collectie), wordt beschreven in een aparte paragraaf, inclusief de eigenschappen van die klasse. Iedere beschrijving bestaat uit 2 delen:
```

Comment:
Vorige week kon ik nog suggestions aanmaken. Nu niet meer...
suggestion: 
"Elk relevant onderdeel..."

Reply:
Raar... ik heb je suggestie in ieder geval overgenomen :-)

Reply:
Sorry @architolk , mijn suggestie was niet volledig. Er staat aan het eind vd zin nog een keer "klasse", dat moet dan ook mee veranderen naar "onderdeel". 

</de muggenzifter>